File: testData/src/java8/pkg/TestTryLoopNoCatch.java
package pkg;

public class TestTryLoopNoCatch {
  public void test(String[] s) {
    boolean b = false;
    for (int i = 0; i < s.length; i++) {
      try {
        b = method(s[i]);
        break;
      } catch (Exception e) {

      }
    }

    System.out.println(b);
  }

  private boolean method(String s) throws Exception {
    if (s.length() > 20) {
      throw new Exception();
    }

    return s.length() > 10;
  }
}

File: plugins/kotlin/testData/src/kt/pkg/TestTryLoopNoCatch.kt
package pkg

class TestTryLoopNoCatch {
    fun test(s: Array<String?>) {
        var b = false
        for (i in s.indices) {
            try {
                b = method(s[i]!!)
                break
            } catch (e: Exception) {
            }
        }

        println(b)
    }

    @Throws(Exception::class)
    private fun method(s: String): Boolean {
        if (s.length > 20) {
            throw Exception()
        }

        return s.length > 10
    }
}
